submission/launch.sh

```diff
@@ -1,19 +1,25 @@
-#!/bin/bash -ex
+#!/bin/bash
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 trap "vagrant destroy -f" EXIT
 
-curl "${ARCHIVE_URL}" -o submission.zip
-curl "${SCRIPT_URL}" -o checker.sh
+curl "${VMCK_ARCHIVE_URL}" -o archive.zip
+curl "${VMCK_SCRIPT_URL}" -o script.sh
+
+touch reuslt.err
 
 vagrant up
-vagrant ssh -- < checker.sh > result.out
+vagrant ssh -- < script.sh 1> result.out 2> result.err
```

Comment:
Since `script.sh` is already in the VM, you could simply write `vagrant ssh -- 'cd /vagrant; ./script.sh' 1>... 2>...` instead of streaming `script.sh` as standard input to ssh. That way, `script.sh` could even be written in Python.